src/main/kotlin/no/nav/klage/domain/KlageAnkeInput.kt

```diff
@@ -79,10 +84,17 @@ data class KlageAnkeInput(
             }
         } else null
     }
-
+    @JsonIgnore
     fun isKlage(): Boolean {
         return klageAnkeType == KlageAnkeType.KLAGE
     }
+
+    val deprecatedFields = listOf(saksnummer)
```

Comment:
Ikke sikkert denne propertien trengs hvis den ikke brukes til noe annet. Kanskje kan brukes on-the-fly e.l.?

Reply:
Nei, gjort om litt her.